mix.exs

```diff
@@ -66,8 +66,7 @@ defmodule Astarte.Core.Generators.MixProject do
       # Test section
       {:dialyxir, "~> 1.4", only: [:dev, :ci], runtime: false},
       {:credo, "~> 1.7", only: [:dev, :test, :ci], runtime: false},
-      {:excoveralls, "~> 0.15", only: :test},
-      {:mox, "~> 0.5", only: :test}
+      {:excoveralls, "~> 0.15", only: [:test, :ci]}
```

Comment:
I would also add `:dev`, so that while developing we can just run `mix coveralls` and get the coverage, without setting explicitly `MIX_ENV`